modules/web/js/ballerina/components/global-definitions.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import './global-definitions.css'
+import ImageUtil from './image-util';
+
+export default class GlobalDefinitions extends React.Component {
+
+    render() {
```

Comment:
looks like `@inheritdoc` optional. If we do not add a tag to a symbol it will inherit doc from parent according to [docs](http://usejsdoc.org/tags-inheritdoc.html). I feel adding this tag for all inherited methods makes the component code a little messy. Maybe we can skip it? WDYT?